.circleci/config.yml

```diff
@@ -221,6 +221,7 @@ jobs:
       - when:
           condition: << parameters.spm >>
           steps:
+            - run: for ip in $(dig @8.8.8.8 github.com +short); do ssh-keyscan github.com,$ip; ssh-keyscan $ip; done 2>/dev/null >> ~/.ssh/known_hosts
```

Comment:
We can remove this workaround once rdar://problem/50686014 is fixed in Xcode or MapboxNavigationNative’s Package.swift points to an HTTPS URL for MapboxCommon, whichever comes first.

Reply:
Removed in #2793, now that mapbox/mapbox-navigation-native-ios#1 has been released.

Reply:
We didn’t end up being able to remove the workaround yet.

Reply:
@1ec5 is that problematic?